webapp/src/DOMJudgeBundle/Utils/Scoreboard/Scoreboard.php

```diff
@@ -377,10 +377,16 @@ public static function scoreTiebreaker(TeamScore $a, TeamScore $b): int
         $btimes = $b->getSolveTimes();
         rsort($atimes);
         rsort($btimes);
-        if (isset($atimes[0])) {
+        if (isset($atimes[0]) && isset($btimes[0])) {
             if ($atimes[0] != $btimes[0]) {
```

Comment:
this condition can be removed: `<=>` will take care of that case.
then also the final `return 0`; can be removed since it's then dead code.